### PR TITLE
feat(sdk): initialize the python version of both client sdk and base sdk

### PR DESCRIPTION
**Pull request by jianzs**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

re #146 

#### 2. What is the scope of this PR (e.g. component or file name):

- Client SDK in Python
- Base SDK in Python
- Quick-start example in Python

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

### CODE REVIEW

File: packages/base-py/pluto_base/utils/resource_id.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import hashlib
+from .configuration import current_project_name, current_stack_name
+
+RESOURCE_ID_MAX_LENGTH = 64
+
+
+def gen_resource_id(*args):
```
Comment (Peefy) [inline packages/base-py/pluto_base/utils/resource_id.py:7]: What does `*args` mean?

File: packages/pluto-py/pluto_client/router.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+from pydantic import BaseModel
+from typing import Callable, Dict, Optional, Union
+from pluto_base.platform import PlatformType
+from pluto_base import utils, resource
+from .clients import shared
+
+
+class HttpRequest:
```
Comment (Peefy) [inline packages/pluto-py/pluto_client/router.py:8]: Consider using the @dataclasses decorator.

File: packages/base-py/pluto_base/utils/resource_id.py (hunk 1 above)
Comment (jianzs) [reply packages/base-py/pluto_base/utils/resource_id.py:7]: There are two ways to utilize this function, each requiring a different number of arguments. This function has been directly translated from TypeScript, available at https://github.com/pluto-lang/pluto/blob/main/packages/base/utils/resource-id.ts . I'll include comments to explain the use of `*args`.

Comment (Peefy) [reply packages/base-py/pluto_base/utils/resource_id.py:7]: Make sense.

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**